### PR TITLE
Batons no longer bypass armor, and arent the best melee anymore

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request

Changes up batons and telebatons so they no longer bypass armor. stun Batons now deal 40 stamina as opposed to 90, and all telebatons deal the same, except contractor, which is 50- enough to barely down someone in two hits. Telebatons no longer knockdown, but their cooldown has been halved. The intent of this is to make batons batons no longer the best melee combat weaponry. They're still more effective than a fireaxe or even an esword (as much damage as the station's highest possible melee weapon roundstart, the claymore, if the ai is an idiot)

Stunbatoning down a heavily armored syndie now takes about five hits

## Why It's Good For The Game
stun and telescopic batons have been the best melee weapons for a good long while, making most non-stunning forms of melee combat somewhat suicidal against opponents who are armed with them. I want to shake up that meta 

~~now the best melee weapon is whip~~

## Changelog
:cl:
balance: stunbatons and telebatons no longer bypass armor
balance: lowers numbers on stun and telebatons
tweak: telebaton has its cooldown halved, but no longer does knockdown
tweak: batons are now far more affected by EMP pulses
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
The vote on this is very controversial, basically even at the moment.

**Comment by zeskorion** [comment]
> The vote on this is very controversial, basically even at the moment.

i'm totally in favor of letting it sit for awhile more, maybe delete the ones you already closed to bring it to the forefront

**Comment by Sun-Soaked** [comment]
Ok so, Zesk, you've expressed the opinion that you don't believe in command and sec having "crutches", and I would appreciate if you elaborate, because I certainly hope you don't actually believe that a roundstart security officer or, for instance, the HoP, should be at the same level as an assistant with a stunprod and toolbox, with the only difference being skill. Obviously skill should always be a factor but *dude*. I don't like sec, and I like a lot of command players even less, but:

-A: Sec and Command are targets, and should thus have premier ways to defend themselves.
-B: Sec and Command are preferred and attractive roles who are in charge of defending both themselves and the station. 
-C: Sec and Command have positive(depending on your point of view) gameplay interactions that are fueled by their preferential combat gear (I.e. constant paranoia as people are thirsting for your shit, constant attacks from antags who want said shit and your access, and elements of constant pressure due to the exclusivity of the role and requirement to make good decisions.)

Roles such as Sec and Command are intended to be forced to defend themselves and others, sometimes constantly, during a round. As a result they have items that make them more powerful in certain fields (nonlethal takedown for sec, close-range nonlethal takedown for command). By your logic of "crutches", why not remove every weapon from the game? Are antags not given a "crutch" via their exclusive items? Are miners not given a combat "crutch" by their anti-megafauna items and abilities? Really, without "crutches" all conflicts in game should certainly be disputed via fisticuffs inside a rage cage by two baseline humans, thus removing all "crutches" and making it truly a test of only skill. The truth is, what you describe as "crutches" when you talk of sec and command having "crutches" you refer to intended roundstart role disparity, as set by the setting and necessary tasks of such peoples. Creating such a complex and engaging gameplay experience REQUIRES inherently unfair interactions, or else risking depleting the unique and complicated qualities of the game. Are there no ways, through skill or ingenuity, to overcome this disparity? There are several for almost every role(Particularly R&D, Cargo, and Engineering roles), not to mention the additional breathing room given to antags via their special powers. The disparities are parts of the games flavour, and trying to iron them out to enforce some element of "fairness" always inevitably results in a previously engaging and interesting faceted interaction being reduced to dull and un-distinct conflict. 
TLDR; Sec and Command may have "crutches" but these are intended game qualities. I tend to dislike the qualities of both roles when playing in opposition to them, but in truth, they make the game a better space to play in.

**Comment by qwertyquerty** [comment]
vote failed

**Comment by dinnerwoerror** [comment]
zeskoranges go home
